src/ActiveLogin.Authentication.BankId.Api/Errors/BankIdApiErrorHandler.cs

```diff
@@ -27,7 +27,7 @@ public static async Task EnsureSuccessAsync(HttpResponseMessage httpResponseMess
             }
         }
 
-        private static async Task<Error> TryGetErrorAsync(HttpResponseMessage httpResponseMessage)
+        private static async Task<Error?> TryGetErrorAsync(HttpResponseMessage httpResponseMessage)
```

Comment:
I know we are in the nullable types PR :) but I would like to use as less as possible nullable occasions. Here we can keep it as not null Error, but within instead of returning null, we can return Error.Empty - no-opt solution.
Implementation inside of Error class could be:
`public static Error Empty => new Error(string.Empty, string.Empty);`
It is much better practice to compare later on.

Reply:
Line 21 here would be:
`if (error != Error.Empty)`
Line 45 here would be 
`return Error.Empty;`

Reply:
Yeah, I like the approach, but don't we expose that "publicly" just to solve an internal thing in this class? From our consumers point of view, we would have an `.Empty` on this model class, but on no other.